boards/risc-v/qemu-rv/rv-virt/src/romfs_stub.c

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * boards/risc-v/qemu-rv/rv-virt/src/romfs_stub.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/compiler.h>
+weak_data const unsigned char aligned_data(4) romfs_img[] =
```

Comment:
add g_ preifx

Reply:
I'm sorry for the late response: I was on leave.

This variable is set by the `genromfs` tool from https://github.com/apache/nuttx-apps/blob/cf27f085f56709ca5e1a31e4a91ca9e90dd69c79/tools/mkromfsimg.sh#L52. We would need to either change the tooling or renaming the variables in the `mkromfsimg.sh` script. This would break other implementations. I don't think it fits this PR.

Reply:
ok